docs/3.0.0-alpha.x/guides/deployment.md

```diff
@@ -867,14 +904,50 @@ Ensure you are logged in as a **non-root** user. You will install **PM2** global
 npm install pm2@latest -g
 ```
 
-Navigate to your **Strapi Project folder** and use the following command to set the environment variable to production and start `pm2`:
+### The ecosystem.config.js file
 
-`Path: ./my-project/`
+- You will need to configure an `ecosystem.config.js` file. This file will manage the **database connection variables** Strapi needs to connect to your database. The `ecosystem.config.js` will also be used by `pm2` to restart your project whenever any changes are made to files within the Strapi file system itself (such as when an update arrives from GitHub). You can read more about this file [here](https://pm2.io/doc/en/runtime/guide/development-tools/).
```

Comment:
You will need to configure an `ecosystem.config.js` file. This file will manage the **database connection variables** that Strapi needs to connect to your database. The `ecosystem.config.js` will also be used by `pm2` to restart your project whenever any changes are made within the Strapi file system itself (such as when an update comes from GitHub). You can read more about this file [here](https://pm2.io/doc/en/runtime/guide/development-tools/).

Reply:
Adding the word "that" isn't really necessary, grammatically. https://www.quickanddirtytips.com/education/grammar/when-to-delete-that - however, if it seems clearer to you, I went ahead and added it.  - After a plural, the word "that" can often be omitted.  

Especially since I was adding a better word "comes" to the paragraph anyway ;)